### PR TITLE
Certify layout tweaks

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#4851

*Description of changes:*

* Updated Layout

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Review by vysakh-menon-aot — APPROVED** [review]

File: src/components/Certify/Certify.vue [hunk 1]
```diff
@@ -125,7 +122,7 @@ export default class Certify extends Vue {
 }
 
 @media (min-width: 768px) {
-  .certifiedby-container {
+  .certifiedBy-container {
```
Comment (severinbeauvais) [inline src/components/Certify/Certify.vue:125]: Should be "certified-by-container".

File: src/components/Certify/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bcrs-shared-components/certify",
-    "version": "1.0.0",
+    "version": "1.0.1",
```
Comment (severinbeauvais) [inline src/components/Certify/package.json:3]: Did you do this or did Lerna?

**Review by severinbeauvais — APPROVED** [review]

File: src/components/Certify/package.json (hunk 2 above)
Comment (cameron-freshworks) [reply src/components/Certify/package.json:3]: Lerna!

File: src/components/Certify/Certify.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/Certify/Certify.vue:125]: copy that

File: src/components/Certify/package.json (hunk 2 above)
Comment (severinbeauvais) [reply src/components/Certify/package.json:3]: Perfect.

File: src/components/Certify/Certify.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/Certify/Certify.vue:125]: I looked it up for fun...
`camelCase`
`PascalCase` aka `TitleCase`
`snake_case`
`kebab-case`

Comment (severinbeauvais) [reply src/components/Certify/Certify.vue]: I don't think there's such as thing as Camel Kebabs 🤣

Comment (cameron-freshworks) [reply src/components/Certify/Certify.vue]: hahah well there is now!! 
😆 Jk jK.

File: src/components/Certify/Certify.vue [hunk 3]
```diff
@@ -136,7 +133,7 @@ export default class Certify extends Vue {
   }
 }
 
-.value.certifiedby {
+.value.certifiedBy {
```
Comment (severinbeauvais) [inline src/components/Certify/Certify.vue:136]: Oops you missed this one.

Comment (cameron-freshworks) [reply src/components/Certify/Certify.vue:136]: 😬  Will get this

File: src/components/Certify/Certify.vue [hunk 4]
```diff
@@ -109,11 +110,7 @@ export default class Certify extends Vue {
   font-size: 0.875rem;
 }
 
-.certify-container {
-  padding: 1.25rem;
-}
-
-.certifiedby-container {
+.certified-by-container {
```
Comment (severinbeauvais) [inline src/components/Certify/Certify.vue:113]: I no longer see this class used in the template. (Also in the media query block below.)

**Pull request merged**